custom_components/easee/translations/en.json

```diff
@@ -37,5 +37,12 @@
       "no_sites": "No monitored sites selected"
     }
   },
+  "system_health": {
+    "info": {
+      "reach_easee_cloud": "Reach Easee Cloud",
```

Comment:
How about "Easee cloud reachable" instead?

Reply:
I saw now that HAs own status information uses the form you used, so it is I guess better to use that.